### ISSUE TITLE
[Feature] Fix the correct document link in Skywalking UI

### ISSUE THREAD

**Issue by JohnDuncan5171** [apache/skywalking#9565]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

About the solutions discussed in [this pr](https://github.com/apache/skywalking/pull/9562#issuecomment-1236537900)

The content we discussed in this link. My intention is not to add all old implementations to all the new features.
Because **all versions** of official documents in UI are linked to the **latest** version.

I find the code and resolution, **perhaps we should link it to the corresponding version official document when we release the new version**
[like this](https://github.com/JohnDuncan5171/skywalking/commit/0e494f2905da7b5ca96f0859ae7568d42e60eec6)

I'm not sure if those JSON files are dynamically created, I'm just determined that the source of the UI link is from [here](https://github.com/apache/skywalking/tree/master/oap-server/server-starter/src/main/resources/ui-initialized-templates) after I checked the [Skywalking UI](https://github.com/apache/skywalking-booster-ui)'s code.

So it's just a demo, I don't want to submit a PR, maybe this change needs to be discussed.

I suggest adding this feature to 9.3.0.
And I also suggest the UI's LOGO or copyright should mark the current version, it will very helpful for new user

### Use case

Every release should be linked to the corresponding version official document when we release the new version.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
We would not simply change that JSON in the runtime or use a variable. Because this JSON is generated from UI's edit mode.

**Comment by JohnDuncan5171** [comment]
Yep. So if those JSON files are dynamically created, we should change the `graph.url`  generator.
like `"url":"https://skywalking.apache.org/docs/main/{version:latest}/en/setup/backend/backend-mysql-monitoring/"`

do our best to provide an accurate document

**Comment by wu-sheng** [comment]
Not that simple. UI is using the same GraphQL query to get the current value, so, when you do so, UI would show `https://skywalking.apache.org/docs/main/{version:latest}/en/setup/backend/backend-mysql-monitoring/` for the link.

**Comment by JohnDuncan5171** [comment]
You are right, of course not that simple. I mean that's just a grammar demo.
I'm not good for the front end. That's why I suggest discussing it.

**Comment by wu-sheng** [comment]
The difficulty is not about changing UI. The thing is, we don't have an API to provide a service like this, and we never think we should make things so complex. Read doc with the version you prefer/need, we assume this is a basic requirement for oss users.

Also, when you take a look Function menu, it even links to a 3rd party page.
 Aliyun gateway team is planning to contribute a dashboard for their gateway service too. If they really do, we have to link it to an Aliyun dev doc page as well.

You can see, as a widely adopted project, this is not a simple tech solution.

**Comment by wu-sheng** [comment]
I am going to convert this to a discussion, nd close this.
For now. we don't have a good solution for above challenges.
